src/robotFactory.js

```diff
@@ -1,10 +1,74 @@
 'use strict';
 
-class BaseRobot {}
+class BaseRobot {
+  constructor(name, weight, coords, chipVersion) {
+    this.name = name;
+    this.weight = weight;
 
```

Comment:
Redundant space here
```suggestion
    this.weight = weight;
```

Reply:
Не понимаю? При npm run lint оно делает пробел между этими 6  и 8 this.